<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmyWidget</class>
 <widget class="QWidget" name="QmyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmyWidget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置男生年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置女生年龄</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoy">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinGirl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBoyInc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>boy长大一岁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGirlInc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>girl长大一岁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClassInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>类的元对象信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>83</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoyInc</sender>
   <signal>clicked()</signal>
   <receiver>spinBoy</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGirlInc</sender>
   <signal>clicked()</signal>
   <receiver>spinGirl</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
